bc_obps/registration/api/v2/_operations/current.py

```diff
@@ -22,9 +22,9 @@
     response={200: List[OperationCurrentOut], codes_4xx: Message},
     tags=["V2"],
     auth=authorize('approved_industry_user'),
-    description="""Gets the list of operations associated with the current user's operation.
-    The endpoint ensures that only authorized industry users can get operations belonging to their operator. Unauthorized access attempts raise an error.""",
+    description="""Gets the list of operations associated with the current user's operation that are not yet registered.
```

Comment:
```suggestion
    description="""Gets the list of operations associated with the current user's operator that are not yet registered.
```

Reply:
(I know this wasn't your boo-boo, I just noticed it now and figured I'd take the opportunity to fix it)